frontend/src/BodyPart.js

```diff
@@ -1,115 +1,80 @@
-              </p>
-              <p className="card-text">
-                <small className="text-muted">Last updated 3 mins ago</small>
-              </p>
-            </div>
-          </div>
-
-          <div className="card">
-            <img className="card-img-top" src={pic5} alt="" />
-            <div className="card-body">
-              <h5 className="card-title">Grilled Chicken with basil</h5>
-              <p className="card-text">
-                “It never fails to blow my mind how a simple marinade, like this
-                rosemary one, can elevate humble ingredients to such a beautiful
-                place.
-              </p>
-              <p className="card-text">
-                <small className="text-muted">Last updated 3 mins ago</small>
-              </p>
-            </div>
-          </div>
-          <div className="card">
-            <img className="card-img-top" src={pic6} alt="" />
-            <div className="card-body">
-              <h5 className="card-title">Salmon with avocado raddish</h5>
-              <p className="card-text">
-                “If you like salmon, you’ll love this quick and easy recipe –
-                think flaky, juicy flesh kissed with gentle spice and the
-                crispiest skin ever, buddied up with my fail-safe, fluffy
-                coconut rice and refreshing cucumber pickle. Get in! ”
-              </p>
-              <p className="card-text">
-                <small className="text-muted">Last updated 3 mins ago</small>
-              </p>
-            </div>
-          </div>
-          <div className="card">
-            <img className="card-img-top" src={pic7} alt="" />
-            <div className="card-body">
-              <h5 className="card-title">Avocado Toast</h5>
-              <p className="card-text">
-                “I’m told that in the UK avocados are now more popular than
-                oranges! This breakfast favourite is a source of vitamin E,
-                which acts as an antioxidant, helping to protect our cells. ”
-              </p>
-              <p className="card-text">
-                <small className="text-muted">Last updated 3 mins ago</small>
-              </p>
-            </div>
-          </div>
+          {recipeList
+            .map((recipe) => (
+              <Link
+                className="card"
+                style={{ width: '100%', overflow: 'hidden' }}
+                to={{
+                  pathname: `/recipes/${recipe.id}`,
+                  state: { ingredient: null },
+                }}
+              >
+                <img
+                  className="card-img-top"
+                  src={`/api/image/${recipe.image.id}`}
+                  alt={`${recipe.image.id}`}
+                  style={{ objectFit: 'cover', height: 217.8 }}
+                />
+                <div className="card-body">
+                  <h5 className="card-title">
+                    {recipe.name.length > 33
```

Comment:
Ez CSS-ből szebben megoldható: https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

Reply:
Kész, viszont csak az alsó kicsi cardokra került fel, mivel a nagy cardokat széthúzta, vagy a szöveget nem tördelte be.